snapcraft/plugins/copy.py

```diff
@@ -65,6 +67,13 @@ def schema(cls):
 
         return schema
 
+    def __init__(self, name, options, project):
+        super().__init__(name, options, project)
+
+        logger.warning("DEPRECATED: The 'copy' plugin's functionality "
+                       "has been replaced by the 'dump' plugin, and it will "
```

Comment:
maybe mention that it's replaces by the dump plugin combined with filesets. Or too much info?